src/qautostart.h

```diff
@@ -23,15 +23,11 @@
 #ifndef AUTOSTART_H
 #define AUTOSTART_H
 
-#include <QObject>
+#include <QString>
```

Comment:
Why not QObject? If you remove QObject, the signal/slot won't work for this class. Any reason removing QObject?

Reply:
You don't use signals, do you? I see it useless because of that. I'll add it if you want though.

Reply:
We may not have any signals now. But How about in future? A library should provide more options for users.

Reply:
Like the constructor, inherit from QObject in the feature if you needed signals and slots. I don't really see how these can be useful in this project though (it doesn't do anything that requires signals and slots).